lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
@dbarashev 

> it is pretty heavy-weight so we probably don't want it to be loaded by default

Are you talking about RequireJS or KaTeX?

So, should KaTeX be loaded when we include a built KaTeX file with `<script>` tag, just like we do in the demo with `ext-language-tools.js`: `<script src="../build/src-noconflict/ext-language_tools.js"></script>`? Or is it fine as is?